### DISCUSSION TITLE
关于海豹增强提案(SEP, Sealdice Enhancement Proposals)的整理

### DISCUSSION THREAD

**Discussion by fy0** [sealdice/sealdice-core#612] [Ideas]
### 在提问之前...

- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我基本确定这是一个新功能/建议，而不是遇到了 bug（不确定的话请附上日志）

### 说说你遇到的问题？

有一些提案性质的issue，应该编号存档，这样可以经常参考

### 有什么好的想法？

仿照pep之类，进行一个sep的编制

**Comment by fy0** [comment]
先列一些issue：
#278 #520 #492

**Comment by Xiangze-Li** [comment]
建议使用 Discussion 和 Pinned Issue。

另外，单独编号恐怕意义不大，因为：

1. Issue 和 Discussion 本身就有编号
2. 记忆编号并不比记忆一个简要标题更简单，尤其是对于只是偶尔来看看的开发者来说
